acetone/utils.py

```diff
@@ -0,0 +1,17 @@
+import importlib
+
+
+def resolve_type(type_name: str, class_separator: str='.'):
```

Comment:
can you tell me a situation where class_separator is anything other than '.'? Maybe I'm not getting the purpose of it